make TensorFlow pick up on build environment defined by EasyBuild
author: Denis Kristak, Pavel Grochal (INUITS)
--- tensorflow-4039feeb743bc42cd0a3d8146ce63fc05d23eb8d/third_party/mlir/tblgen.bzl.orig	2021-08-13 16:18:12.735593126 +0200
+++ tensorflow-4039feeb743bc42cd0a3d8146ce63fc05d23eb8d/third_party/mlir/tblgen.bzl	2021-08-13 16:18:48.615675683 +0200
@@ -173,6 +173,7 @@
         inputs = trans_srcs,
         executable = ctx.executable.tblgen,
         arguments = [args],
+        use_default_shell_env = True,
         mnemonic = "TdGenerate",
     )
 
